<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recommend</class>
 <widget class="QWidget" name="recommend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(225, 255, 253);

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="PanelItem" name="panelItem">
     <property name="titleText">
      <string>距离优先</string>
     </property>
     <widget class="QTextEdit" name="textEdit_distance">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>391</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color:transparent;
color:rgb(17,17,17);
selection-background-color:rgb(236,116,64);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="PanelItem" name="panelItem_2">
     <property name="titleText">
      <string>价格优先</string>
     </property>
     <widget class="QTextEdit" name="textEdit_price">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>391</width>
        <height>711</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color:transparent;
color:rgb(17,17,17);
selection-background-color:rgb(236,116,64);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="PanelItem" name="panelItem_3">
     <property name="titleText">
      <string>智能推荐</string>
     </property>
     <widget class="QTextEdit" name="textEdit_smart">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>381</width>
        <height>711</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color:transparent;
color:rgb(17,17,17);
selection-background-color:rgb(236,116,64);
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelItem</class>
   <extends>QWidget</extends>
   <header>panelitem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
